pkg/gui/branches_panel.go

```diff
@@ -99,6 +99,19 @@ func (gui *Gui) handleCreatePullRequestPress(g *gocui.Gui, v *gocui.View) error
 	return nil
 }
 
+func (gui *Gui) handleCopyPullRequestURLPress(g *gocui.Gui, v *gocui.View) error {
+	pullRequest := commands.NewPullRequest(gui.GitCommand)
+
+	branch := gui.getSelectedBranch()
+	if err := pullRequest.CopyURL(branch); err != nil {
+		return gui.surfaceError(err)
+	}
+
+	return gui.createPopupPanel(createPopupPanelOpts{
+		prompt: "Pull request URL copied to clipboard",
```

Comment:
We'll want to use `gui.Tr` here.

Also I've been intending to add a way to have a message show up for a few seconds in the bottom right, where you'd typically see a message with a loader while an operation is running. I've got a WIP for that on another branch. Is that something you'd be okay with (of course I'm happy to merge this PR using the prompt)? Having the prompt come up is more obvious but also requires user input to dismiss which might get tiresome if you're copying things to the clipboard a lot.

Reply:
Good point, I missed that :slightly_smiling_face: 

About the message, I'm good with the message being shown there. However, in the case of copying a PR URL, I guess it doesn't happen that often, so it wouldn't be that bad to display it as is. On the other hand, it would be more consistent if all the "clipboard messages" are shown in the same place.

You are the boss here :wink: , so I'll leave it up to you :slightly_smiling_face:

Reply:
Happy to merge this with the prompt appearing but I'll make another issue for adding the new 'toast' thingo so that we can standardise it :)

Reply:
Perfect! That's good with me. You can add me to the issue when you create it :slightly_smiling_face: